app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```diff
@@ -92,30 +104,52 @@ func (s *ForwardingRuleSyncer) EnsureHttpForwardingRule(lbName, ipAddress, targe
 }
 
 func (s *ForwardingRuleSyncer) DeleteForwardingRules() error {
-	// TODO(nikhiljindal): Also delete the https forwarding rule when we start creating it.
+	var err error
 	name := s.namer.HttpForwardingRuleName()
-	fmt.Println("Deleting forwarding rule", name)
-	err := s.frp.DeleteGlobalForwardingRule(name)
-	if err != nil {
-		fmt.Println("error", err, "in deleting forwarding rule", name)
-		return err
+	fmt.Println("Deleting http forwarding rule", name)
+	httpErr := s.frp.DeleteGlobalForwardingRule(name)
+	if httpErr != nil {
+		httpErr := fmt.Errorf("error %s in deleting http forwarding rule %s", httpErr, name)
+		fmt.Println(httpErr)
+		err = multierror.Append(err, httpErr)
+	} else {
+		fmt.Println("Http forwarding rule", name, "deleted successfully")
 	}
-	fmt.Println("forwarding rule", name, "deleted successfully")
-	return nil
+	name = s.namer.HttpsForwardingRuleName()
+	fmt.Println("Deleting https forwarding rule", name)
+	httpsErr := s.frp.DeleteGlobalForwardingRule(name)
+	if httpsErr != nil {
```

Comment:
you said: just look for NotFound...

Reply:
Am surprised why didnt the tests fail.
DeleteGlobalForwardingRule seems to be masking the NotFound error.
Ref https://github.com/kubernetes/kubernetes/issues/54717.

Updated the code to check for NotFound